modules/02b-cdc-connect-debezium.md

```diff
@@ -0,0 +1,120 @@
+# Module 02 - CDC with Apache Kafka Connect and Debezium
+
+## Deploy and check PostgreSQL database
+
+Deploy the PostgreSQL database used as storage for devices information with some pre-crated data.
+
+```shell
+oc adm policy add-scc-to-user anyuid -z postgres
+oc apply -f kafka-connect-debezium/postgres/postgres.yaml
+```
+
+Check that the table `deviceinfo` is prepopulated with some devices related information.
+
+```shell
+oc exec $(oc get pods --selector=app=postgres -o=jsonpath='{.items[0].metadata.name}') -- env PGOPTIONS="--search_path=devices" psql -U postgres -c "SELECT * FROM deviceinfo;"
+```
+
+## Deploy Apache Kafka Connect
+
+The Apache Kafka Connect cluster can be deployed using the cluster operator.
+There are two available CRDs (Custom Resource Definitions) for that: `KafkaConnect` and `KafkaConnectS2I`.
+The `KafkaConnect` CRD uses Kafka Connect image as a base layer and the connector plugins are injected by building new image on top of it.
+The `KafkaConnectS2I` CRD leverages the Source-2-Image OpenShift's feature for handling builds and adding connectors plugin.
+In this workshop we will use `KafkaConnect` for adding the Debezium PostgreSQL plugin connector.
+
+First, we will need to build a image containing desired plugins.
+NOTE: You can skip building image and use prepared one from the `sknot/debezium-connector-postgres:0.9.5.Final`
+
+To download debezium connector plugin, build and push image to `your_repository` you can use a command:
+```shell
+export DEBEZIUM_VERSION=0.9.5.Final
+mkdir -p kafka-connect-debezium/my-plugins && cd kafka-connect-debezium/my-plugins && \
+curl http://central.maven.org/maven2/io/debezium/debezium-connector-postgres/$DEBEZIUM_VERSION/debezium-connector-postgres-$DEBEZIUM_VERSION-plugin.tar.gz | tar xz && cd .. && \
+docker build -t your_repository/debezium-connector-postgres:$DEBEZIUM_VERSION . && \
+docker push your_repository/debezium-connector-postgres:$DEBEZIUM_VERSION  && \
+rm -rf my-plugins && cd ..
+```
+
+You can deploy the Kafka Connect cluster with the built image now.
+
+```shell
+oc apply -f kafka-connect-debezium/kafka-connect.yaml
+```
+NOTE: If you built your own image, you have to replace it in the `spec.image`.
```

Comment:
you should be clear that this step has to be done in the `KafkaConnect` resource YAML